webapp/src/components/preview-movies/style/row/Arrow.tsx

```diff
@@ -1,19 +1,19 @@
 import React from 'react';
-import '../../../../../public/assets/css/arrow.css';
 import classNames from 'classnames';
+import '../../../../../public/assets/css/arrow.css';
+import { ChevronRight, ChevronLeft } from 'react-feather';
 
 type Props = {
   left?: boolean,
   right?: boolean,
```

Comment:
plutôt que 2 props, je te propose de faire ainsi :

```suggestion
  orientation: 'left' | 'right'
```

Tu peux également utiliser une Enum, mais ça t'oblige à l'exporter, c'est plus verbeux.